### PR TITLE
feat: Removal of `/config/bridge/` endpoint (#4589)

### LINKED ISSUE #4589: Remove the /config/bridge/ endpoint
## Action Item
The REST endpoint GET and POST `/config/bridge/` returns the username and password used to login to the bridge. 

**Task**:

- Remove the endpoint from the backend
- Remove the CLI `-o` flag from `configure bridge`

### Definition of Done
 
- [x]  `keptn configure bridge --output` does not output any credentials, but just gives the user a hint how to get the credentials using `kubectl`
- [x] Remove the `/config/bridge/` endpoint
- [ ] Update the [documentation](https://keptn.sh/docs/0.8.x/operate/install/#authenticate-keptn-bridge) to set up a new user
- [x] Adapt integration tests (look where `keptn configure bridge --output` or the endpoint is directly used)

### PR DESCRIPTION
**Pull request by warber**
Closes #4589

This PR removes the `GET` and `POST` endpoint to configure bridge credentials. The endpoint were removed from the api module as well as from the swagger spec:

image

`keptn configure bridge ...` command is still there, however it just outputs hints how to read or edit these credentials using `kubectl`.

Further, the `keptn status` command was extended to also output the URL of the Keptn bridge.

### CODE REVIEW

**Comment by warber** [comment]
integration tests from this branch still needs to be triggered.

**Review by agrimmer — COMMENTED** [review]
Can you please also adapt the RBAC rules https://github.com/keptn/keptn/blob/master/installer/manifests/keptn/charts/control-plane/templates/rbac.yaml#L279 ?

File: installer/manifests/keptn/charts/control-plane/templates/rbac.yaml [hunk 1]
```diff
@@ -252,46 +252,6 @@ subjects:
...
-    app.kubernetes.io/name: keptn-api-service-manage-bridge-secret
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
-    app.kubernetes.io/part-of: keptn-{{ .Release.Namespace }}
-    app.kubernetes.io/component: {{ include "control-plane.name" . }}
-    helm.sh/chart: {{ include "control-plane.chart" . }}
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: Role
-  name: keptn-manage-bridge-secret
```
Comment (agrimmer) [inline installer/manifests/keptn/charts/control-plane/templates/rbac.yaml:270]: You did not delete the role keptn-manage-bridge-secret and keptn-delete-bridge-pod. Is this role used somewhere else? If no, please also delete the role.

Comment (warber) [reply installer/manifests/keptn/charts/control-plane/templates/rbac.yaml:270]: removed

File: api/models/error.go [hunk 2]
```diff
@@ -51,6 +53,11 @@ func (m *Error) validateMessage(formats strfmt.Registry) error {
 	return nil
 }
 
+// ContextValidate validates this error based on context it is used
```
Comment (mowies) [inline api/models/error.go:56]:
```suggestion
// ContextValidate validates this error based on context it is used in
```
word missing? same for other occurences

File: cli/cmd/configure_bridge.go [hunk 3]
```diff
@@ -35,145 +24,24 @@ var bridgeCmd = &cobra.Command{
...
-	if err != nil {
-		return errors.New("Could not configure bridge: " + err.Error())
-	}
-	fmt.Println("Bridge credentials configured successfully")
-	return nil
-}
-
-func verifyConfigureBridgeParams(configureBridgeParams *configureBridgeCmdParams) error {
-	if configureBridgeParams.Read != nil && *configureBridgeParams.Read {
+		fmt.Println("This command is not supported anymore!")
```
Comment (mowies) [inline cli/cmd/configure_bridge.go:28]: I think it would make sense to add the version that deprecated this command here

**Review by mowies — CHANGES_REQUESTED** [review]

File: api/models/error.go (hunk 2 above)
Comment (warber) [reply api/models/error.go:56]: the file(s) are generated by the swagger tool ¯\_(ツ)_/¯

File: cli/cmd/configure_bridge.go (hunk 3 above)
Comment (warber) [reply cli/cmd/configure_bridge.go]: done

**Review by bacherfl — APPROVED** [review]
lgtm

**Review by mowies — APPROVED** [review]

**Pull request merged**